test/e2e/user_management_test.go

```diff
@@ -44,80 +44,40 @@ func (s *userManagementTestSuite) TearDownTest() {
 func (s *userManagementTestSuite) TestUserDeactivation() {
 	s.hostAwait.UpdateHostOperatorConfig(test.AutomaticApproval().Enabled())
 
-	s.T().Run("deactivate a user", func(t *testing.T) {
+	s.T().Run("verify user deactivation on each member cluster", func(t *testing.T) {
 		// Initialize metrics assertion counts
 		metricsAssertion := InitMetricsAssertion(s.T(), s.hostAwait)
 
-		userSignup, mur := s.createAndCheckUserSignup(true, "usertodeactivate", "usertodeactivate@redhat.com", true, ApprovedByAdmin()...)
+		// User on member cluster 1
+		userSignup, mur := s.createAndCheckUserSignup(true, "usertodeactivate", "usertodeactivate@redhat.com", s.memberAwait, ApprovedByAdmin()...)
+
+		// User on member cluster 2
+		userSignupMember2, murMember2 := s.createAndCheckUserSignup(true, "usertodeactivate", "usertodeactivate@redhat.com", s.memberAwait, ApprovedByAdmin()...)
```

Comment:
Shouldn't it use the other `memberAwait2` so the user is provisioned to the second member cluster? That was the purpose of this change, right?

Reply:
🤦  yes! Let me fix that.

Reply:
Fixed https://github.com/codeready-toolchain/toolchain-e2e/pull/226/commits/c078300ee00dc8155b167fefbc24bf06c9942825